src/composables/UserAuthentication.js

```diff
@@ -46,14 +48,64 @@ export function setUserManager(providerName) {
   });
 }
 
+/**
+ * Set the current user's session token in the local storage.
+ * @param {string} sessionToken The session token to store.
+ */
+export function setUserSessionToken(sessionToken) {
+  localStorage.setItem('sessionToken', sessionToken);
+}
+
+/**
+ * Get the current user's session token from the local storage.
+ * @returns {string} the session token of user from the local storage.
+ */
+export function getUserSessionToken() {
+  return localStorage.getItem('sessionToken');
+}
+
 /**
  * Initiate the login process by redirecting the user to the authentication provider.
  * @returns {Promise<void>} Promise with nothing on success otherwise an error.
  */
-export async function login() {
+export async function logine() {
```

Comment:
logine ???

Reply:
think is useless code

Reply:
already deleted in the correct commit

Reply:
You can't leave this in commit. You have to delete this function in this commit (Add new way of authenticate the user). In doing that, github will show us that you have modified the content of function and its documenation. That would be better.